02-peer-protocol.md

```diff
@@ -101,25 +102,63 @@ desire to set up a new channel.
-
-`max_htlc_value_in_flight_msat` is a cap on total value of outstanding HTLCs, which allows a node to limit its exposure to HTLCs; similarly, `max_accepted_htlcs` limits the number of outstanding HTLCs the other node can offer. `channel_reserve_satoshis` is the minimum amount that the other node is to keep as a direct payment. `htlc_minimum_msat` indicates the smallest value HTLC this node will accept.
-
-`feerate_per_kw` indicates the initial fee rate by 1000-weight (i.e. 1/4 the more normally-used 'feerate per kilobyte') which this side will pay for commitment and HTLC transactions, as described in [BOLT #3](03-transactions.md#fee-calculation) (this can be adjusted later with an `update_fee` message). `to_self_delay` is the number of blocks that the other nodes to-self outputs must be delayed, using `OP_CHECKSEQUENCEVERIFY` delays; this is how long it will have to wait in case of breakdown before redeeming its own funds.
+The `temporary_channel_id` is used to identify this channel until the
+funding transaction is established. 
+
+`funding_satoshis` is the amount the sender is putting into the
+channel. `push_msat` is an amount of initial funds that the sender is
+unconditionally giving to the receiver. `dust_limit_satoshis` is the
+threshold below which outputs should not be generated for this node's
+commitment or HTLC transactions (i.e. HTLCs below this amount plus
+HTLC transaction fees are not enforceable on-chain). This reflects the
+reality that tiny outputs are not considered standard transactions and
+will not propagate through the Bitcoin network. `channel_reserve_satoshis` 
+is the minimum amount that the other node is to keep as a direct
+payment. `htlc_minimum_msat` indicates the smallest value HTLC this
+node will accept.
+
+`max_htlc_value_in_flight_msat` is a cap on total value of outstanding
+HTLCs, which allows a node to limit its exposure to HTLCs; similarly,
+`max_accepted_htlcs` limits the number of outstanding HTLCs the other
+node can offer. 
+
+`feerate_per_kw` indicates the initial fee rate by 1000-weight
+(i.e. 1/4 the more normally-used 'feerate per kilobyte') that this
+side will pay for commitment and HTLC transactions, as described in
+[BOLT #3](03-transactions.md#fee-calculation) (this can be adjusted
+later with an `update_fee` message). 
+
+`to_self_delay` is the number of blocks that the other nodes to-self
+outputs must be delayed, using `OP_CHECKSEQUENCEVERIFY` delays; this
+is how long it will have to wait in case of breakdown before redeeming
+its own funds.
+
+`funding_pubkey` is the public key in the 2-of-2 multisig script of
+the funding transaction output.
+
+The various `_basepoint` fields are used to [derive unique
+keys](03-transactions.md#key-derivation) for each commitment
```

Comment:
We've been using specific references to other BOLTs throughout for consistency, i.e. 'as described in [BOLT 3](...key-derivation)'. Maybe this should be added to stylesheet?

Reply:
Changed "to derive unique keys as described in [BOLT #3](03-transactions.md#key-derivation)".